crux_core/src/typegen.rs

```diff
@@ -8,15 +8,15 @@
 //!
 //! Ensure that you have the following line in the `Cargo.toml` of your `shared_types` library.
 //!
-//! ```rust
+//! ```rust,ignore
```

Comment:
Doctests in this file started failing when I added some regular tests, so they had to be ignored.